src/Quiz/Quiz.js

```diff
@@ -11,62 +71,68 @@ const Quiz = (props) => {
           type="text"
           placeholder="e.g. Cancun 20 08!"
           name="name"
+          onChange={handleChange}
         />
       </div>
       <div className="trip-gender-field">
         <h2>2. I'm packing for:</h2>
-        <button>Female</button>
-        <button>Male</button>
-        <button>Non-Binary</button>
+        <button name='female' onClick={(event) => handleClick(event, 'gender')}>Female</button>
+        <button name='male' onClick={(event) => handleClick(event, 'gender')}>Male</button>
+        <button name='non-binary' onClick={(event) => handleClick(event, 'gender')}>Non-Binary</button>
       </div>
       <div className="trip-location-field">
         <h2>3. Going to:</h2>
         <input
           type="text"
           placeholder="e.g. Mexico"
-          name="location"
+          name="destination"
+          onChange={handleChange}
         />
       </div>
       <div className="trip-duration-field">
         <h2>4. Number of days:</h2>
         <input 
           type="number"
           placeholder="e.g. 3"
-          name="days"
+          name="number_of_days"
+          onChange={handleChange}
+          min="1"
         />
       </div>
       <div className="trip-weather-field">
         <h2>5. What will the weather be like?</h2>
         <p className="select-all-txt">(Select all that apply)</p>
         <div className="quiz-answer-btns">
-          <button>Cold</button>
-          <button>Cool</button>
-          <button>Warm</button>
+          <button name='cold' onClick={(event) => handleClick(event, 'weather')}>Cold</button>
```

Comment:
We need to remember to add the `%` before and after these before we make the API call.